From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Yousong Zhou <yszhou4tech@gmail.com>
Date: Thu, 29 Feb 2024 14:36:49 +0800
Subject: [PATCH 1/2] Fix E1312 when quitting a window

The idea is taken from README.md of MattLombana/dotfiles

Fixes https://github.com/preservim/tagbar/issues/851
---
 autoload/tagbar.vim | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/autoload/tagbar.vim b/autoload/tagbar.vim
index 93cad31..d8de771 100644
--- a/autoload/tagbar.vim
+++ b/autoload/tagbar.vim
@@ -3541,7 +3541,7 @@ function! s:HandleOnlyWindow() abort
 
         try
             try
-                quit
+                call timer_start(0, {-> execute('q', 'silent!') })
             catch /.*/ " This can be E173 and maybe others
                 call s:OpenWindow('')
                 echoerr v:exception
